packages/cli/src/util/is-environment-config.ts

```diff
@@ -0,0 +1,17 @@
+interface EnvironmentConfig {
+  pf_stack_version?: string;
+  sla_target?: number;
+}
+
+export function isEnvironmentConfig(
+  value: unknown
+): value is EnvironmentConfig {
+  if (typeof value !== "object" || value === null) return false;
+
+  // Check if the property exists, and if it does, ensure it's a string
+  const config = value as Record<string, unknown>;
+  return !(
+    "pf_stack_version" in config &&
```

Comment:
Not sure what the purpose of this function is exactly, but this doesn't seem to be the correct logic. `pf_stack_version` doesn't necessarily have to be in the `environment.yaml`.

Reply:
I think instead you want to write an analog to `pf-get-terragrunt-variables` and use that.